keyboards/latinpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,70 @@
+ /* Keymap _0: (Base Layer) Default Layer
+
+   * .-----------------------.
+   * |  7  |  8  |  9  | MO1 |
+   * |-----|-----|-----|-----|
+   * |  4  |  5  |  6  |  +  |
+   * |-----|-----|-----|-----|
+   * |  1  |  2  |  3  |  —  |
+   * |-----|-----|-----|-----|
+   * |  0  |  .  | DEL |Enter|
+   * .-----------------------.
+   * |PGUP | PGDN| 
+   * .-----------.
+   */
+     /* Keymap _1: (Second Layer) second Layer
+
+   * .---------------------------------.
+   * |RGB_TOG|RGB_MOD|RGB_M_K|RGB_M_X |
+   * |-------|-------|-------|--------|
+   * |RGB_SAI|RGB_SAD|RGB_HUI|RGB_HUD |
+   * |-------|-------|-------|--------|
+   * |RGB_VAI|RGB_VAD|RGB_SPI|RGB_SPD |
+   * |-------|-------|-------|--------|
+   * |RGB_SPI|RGB_SPD|RGB_TOG|KC_TRNS |
+   * .--------------------------------.
+   * |NUMLOCK|Caculator | 
+   * .------------------.
```

Comment:
Any mock-ups of keymaps or the keymap's keycode assignments themselves should resemble the assembled hardware.

```suggestion
   * .-----------.
   * |PGUP | PGDN| 
   * |-----------------------.
   * |  7  |  8  |  9  | MO1 |
   * |-----|-----|-----|-----|
   * |  4  |  5  |  6  |  +  |
   * |-----|-----|-----|-----|
   * |  1  |  2  |  3  |  —  |
   * |-----|-----|-----|-----|
   * |  0  |  .  | DEL |Enter|
   * .-----------------------.
   */
     /* Keymap _1: (Second Layer) second Layer

   * .---------------.
   * |NUMLOCK|Calc.  | 
   * |--------------------------------.
   * |RGB_TOG|RGB_MOD|RGB_M_K|RGB_M_X |
   * |-------|-------|-------|--------|
   * |RGB_SAI|RGB_SAD|RGB_HUI|RGB_HUD |
   * |-------|-------|-------|--------|
   * |RGB_VAI|RGB_VAD|RGB_SPI|RGB_SPD |
   * |-------|-------|-------|--------|
   * |RGB_SPI|RGB_SPD|RGB_TOG|KC_TRNS |
   * .--------------------------------.
```